helloworld/service/frontend/BUILD

```diff
@@ -4,6 +4,7 @@
 python_sources(
     name="lib",
     dependencies=[
+        "//:requirements#django-debug-toolbar",
```

Comment:
I think it would be better to add the `django-debug-toolbar` to `python_requirements(module_mapping={.....})` in this project and then let pants string based dependency inference do its thing.
I think that way is more idiomatic.

Reply:
Good idea. The module mapping is already in the default, so just removing this line is fine.